fiftyone/plugins/definitions.py

```diff
@@ -13,7 +13,7 @@
 import eta.core.serial as etas
 
 import fiftyone as fo
-import fiftyone.plugins.constants as constants
+import fiftyone.plugins.constants as fpc
```

Comment:
This style is different in the other files. Change the others to be consistent.

Reply:
[fixed](https://github.com/voxel51/fiftyone/pull/5333/commits/75c481b9dbae6e96482a7d81a152c333b6775341)